backend/uclapi/oauth/urls.py

```diff
@@ -4,7 +4,6 @@
 
 urlpatterns = [
     url(r'myapps/shibcallback$', views.myapps_shibboleth_callback),
-	url(r'myapps', views.my_apps),
```

Comment:
Why is this being removed?

Reply:
Moving the settings page from /oauth/myapps => /settings because it makes more sense logically based on our navigation